packages/verification-vue-component/build/rollup.config.js

```diff
@@ -4,6 +4,7 @@ import resolve from '@rollup/plugin-node-resolve'
 import commonjs from '@rollup/plugin-commonjs'
 import babel from 'rollup-plugin-babel'
 import pkg from '../package.json'
+import wasm from '@rollup/plugin-wasm'
```

Comment:
This new dependency is not in certifaction-web.  Should we also back port it there later?

Reply:
As per my understanding, this "rollup" is only used to package the modules here and is not needed for those who use it like certifaction-web.